### PR TITLE
Added CNCF Project List Link and enhancing the documentation

### PR DESCRIPTION
**Pull request by NitishKumar06**
Problem:
This is a Feature Request

What would you like to be added
Update https://kubernetes.io/docs/contribute/ so that early in the page there is a link to
https://contribute.cncf.io/contributors/projects/#kubernetes

You should decide on some suitable text for the hyperlink. Read https://contribute.cncf.io/contributors/ to get ideas about what to write, if you're not sure.

ℹ️ Don't remove any of the existing links; this is an addition.

Solution:
I've kept the description short for the newbie to come and interact with the CNCF project site, thus adding features.

### CODE REVIEW

**Comment by NitishKumar06** [comment]
Hey @sftim  and @shannonxtreme , Please go through the changes I've made and feel free to suggest some changes if required.

**Comment by NitishKumar06** [comment]
@sftim What's branch conflicts and how can it be resolved?

File: content/en/docs/tasks/administer-cluster/certificates.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Certificates
+title: Generating Certificates Manually
 content_type: task
 weight: 20
 ---
```
Comment (Sea-n) [inline content/en/docs/tasks/administer-cluster/certificates.md:5]: Why do you proposing this change? It seems like from another PR.

**Comment by Sea-n** [comment]
I think GitHub clearly point it out that your local branch is 35 commits behind our main branch.
Please do a `git rebase` before submitting your PR.

image

**Review by Sea-n — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
/close
We are not accepting any PRs against #33848 due to bad conduct.
(ref: [Slack message from @divya-mohan0209, co-chair, SIG-Docs](https://kubernetes.slack.com/archives/C1J0BPD2M/p1653309211262869?thread_ts=1653307256.584269&cid=C1J0BPD2M))

**Pull request closed without merging**